modules/analysis/src/endpoints/test.rs

```diff
@@ -613,3 +613,45 @@ async fn issue_tc_2054(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
 
     Ok(())
 }
+
+#[test_context(TrustifyContext)]
+#[test(actix_web::test)]
+async fn issue_spdx_package_of(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
```

Comment:
another nit, inspired by you, actually :wink: 
```suggestion
async fn spdx_package_of(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
```